frontend/components/SearchBar.vue

```diff
@@ -5,30 +5,18 @@
   >
     <Icon name="bi:search" size="1em" class="flex-shrink-0 w-4 h-4 my-1" />
     <Transition name="search">
-      <input
-        v-if="sidebar.collapsed == false || sidebar.collapsedSwitch == false"
-        ref="input"
-        class="w-16 h-5 bg-transparent outline-none"
-        type="text"
-        placeholder="Search"
-        @focus="onFocus"
-        @blur="onFocusLost"
-        :class="{ 'focus:w-5/6': isInputFocused }"
-      />
+      <input v-if="sidebar.collapsed == false || sidebar.collapsedSwitch == false" ref="input"
+        class="w-16 h-5 bg-transparent outline-none" type="text" placeholder="Search" @focus="onFocus" @blur="onFocusLost"
+        :class="{ 'focus:w-5/6': isInputFocused }" />
     </Transition>
     <Transition name="shortcuts">
-      <div
-        v-if="sidebar.collapsed == false || sidebar.collapsedSwitch == false"
-        ref="hotkeyIndicators"
-        class="absolute right-0 flex pr-6 space-x-1 transition-opacity transition-duration-200"
-      >
+      <div v-if="sidebar.collapsed == false || sidebar.collapsedSwitch == false" ref="hotkeyIndicators"
+        class="absolute right-0 flex pr-2 space-x-1 transition-opacity transition-duration-200">
```

Comment:
I think that this should be maintained at pr-6, but it's also mainly because of a change I sent through in the interim. I remember that we were talking about making this relative to the contianer, but then it makes it so that the tooltips get cut off at the edge of where the searchbar's margins intersect with the sidebar header.

If you have another suggestion I'd be happy to consider it!

Reply:
This is what the tooltip looks like for me locally with `relative` in the top div that I removed from here and pr-2:

![Screenshot_2023-08-12 01 06 54](https://github.com/activist-org/activist/assets/24387426/df215fe3-ccd0-4015-8a32-059ca1af4098)